### ISSUE TITLE
Mouse wheel scrolling disabled when mouse over listbox

### ISSUE THREAD

**Issue by petrihakkinen** [ocornut/imgui#1681]
I'm wondering if it would be possible to not disable mouse wheel scrolling when mouse is over a listbox that does not have a scrollbar? Please see the following screenshot from the Shinobi editor. Currently mouse wheel scrolling is arduous because the scrolling stops when mouse hovers over the listbox, even though the listbox itself does not have a scrollbar. Could the mouse wheel input be redirected to the parent window's scrollbar in this case?

![imgui_listbox](https://user-images.githubusercontent.com/2853615/37509219-28d9f0c8-28ff-11e8-8819-5e77214410bc.png)

**Comment by ocornut** [comment]
The ListBox api are among the last I need to rework/refactor similarly to what has been done with `BeginCombo()`. Basically you can think of `ListBoxHeader()/ListBoxFooter()` as potential `BeginListBox()/EndListBox()` but when I get around to actually rename them I'll have to reevaluate their API (and will keep the old one as legacy api).

ListBoxFooter is currently only pretty much doing some weird hacky mumbo jumbo that can be simplified, but you'll find that essentially you could be able to just call: `BeginChildFrame()` / `EndChildFrame()`.

And then you can pass the `ImGuiWindowFlags_NoScrollWithMouse` flag to `BeginChildFrame()`.
And you'll find that `BeginChildFrame()` is basically some style changes + `BeginChild`.
In other hand you really don't need ListBox at all :)

**Comment by petrihakkinen** [comment]
Ok, thanks for the answer! To clarify, can I get the desired behavior with the current API (e.g. via some BeginChildFrame stuff you mentioned) or do I need to wait for the rework you mentioned? I don't mean to rush, just curious if I can work around the issue?

**Comment by ocornut** [comment]
Yes with BeginChildFrame and that flag it will disable the scrolling and forward it to the parent window.

**Comment by petrihakkinen** [comment]
Ok, thanks! I'll try it.

**Comment by ocornut** [comment]
@petrihakkinen Just to confirm if that worked well for you?
I may be able to look at ListBox API soon-ish so gathering todos/feedback there.

**Comment by ocornut** [comment]
Was browsing ancient issues and realized this one has been solved for a while.
(I don't exactly remember when/where, but we made is that windows not requiring scroll are not polling the scroll wheel, and that was a while ago)
